README.md

```diff
@@ -15,7 +14,7 @@ Install the [Origami build tools](https://github.com/Financial-Times/origami-bui
 
 ## Usage
```

Comment:
i think you probably wanted to remove this heading ^_^

Reply:
I did!